flask_rebar/rebar.py

```diff
@@ -548,6 +554,12 @@ def register(self, app):
         self._register_swagger_ui(app=app)
 
     def _register_routes(self, app):
+        if self.handlers is not None:
+            handlers = self.handlers if isinstance(self.handlers, list) else [self.handlers]
```

Comment:
We should probably handle the string case in the `__init__`, that would simplify the logic anytime we use self.handlers as it will then always be `Optional[List[str]]`

Reply:
you know i did that the first time, and then got an error because `[None]` is not None, and i ended up with this. I have a better solution though.